### DISCUSSION TITLE
Можно ли заменить все поддомены одним доменом с символьной ссылкой?

### DISCUSSION THREAD

**Discussion by NotMephisto** [bol-van/zapret#919] [Q&A] [answered]
Вместо `my.sufrshark.com`, `ux.surfshark.com` и т.п. лишь заменить одной строкой `*.surfshark.com`, чтобы он постоянно не добавлял новые поддомены и знал что нужно обходить все ссылки содержащие домен `surfshark.com` или другого сайта/домена

**Comment by bol-van** [comment, marked as answer]
.* не надо
пишите домен, все поддомены автоматом учитываются
